packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -325,6 +327,11 @@ - (BOOL)initializeUpdatesDatabaseWithError:(NSError ** _Nullable)error
   return dbError == nil;
 }
 
+- (void)purgeUpdatesLogsOlderThanOneDay
+{
+  [EXUpdatesUtils purgeUpdatesLogsOlderThanOneDay];
```

Comment:
inline this at callsite, no need for the separate method EXUpdatesAppController.purgeUpdatesLogsOlderThanOneDay just to have this

Reply:
I prefer this